drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
+  labels:
```

Comment:
@prydonius @migmartri apart from querying secrets based on labels, these labels are pretty useless. Should we get rid of them?